components/DropzoneComponent.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { DropzoneOptions, useDropzone } from 'react-dropzone';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+
+interface DropzoneComponentProps extends DropzoneOptions {
+  onChange: (file: File | null) => void;
+}
+
+const DropzoneComponent = ({
+  onChange,
+  accept,
+  maxSize,
+  minSize,
+  multiple,
+  ...rest
+}: DropzoneComponentProps) => {
+  const onHandleDrop = (acceptedFiles: File[]) => {
+    onChange(acceptedFiles.length > 0 ? acceptedFiles[0] : null);
```

Comment:
A slightly better approach would be to take into consideration `multiple` property. If it's `true`, then we use all accepted files. If `false`, then we only take the first.